server/utils/db/connection_pool.py

```
@@ -111,6 +142,29 @@ def get_connection(self):
+                with connection.cursor() as cur:
+                    if user_id:
+                        cur.execute("SET myapp.current_user_id = %s", (user_id,))
+                    if org_id:
+                        cur.execute("SET myapp.current_org_id = %s", (org_id,))
+            elif not request.path.startswith("/health"):
+                logger.warning(
+                    "No user_id or org_id available in request context for %s %s ",
+                    request.method, request.path,
+                )
+        except Exception:
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)